2.0/contrib/src/.bashrc

```diff
@@ -0,0 +1,2 @@
+# This will make scl collection binaries work out of box.
+source scl_source enable ruby200 ror40
```

Comment:
Yes that's correct, and if you create `./contrib/src/.bashrc` it will be copied to `/opt/openshift`. Which will result in `.bashrc` land in `/opt/openshift/src` which is `$HOME` dir for the user. It's the same as in https://github.com/openshift/sti-python.